server/raft.go

```diff
@@ -2084,7 +2088,7 @@ func (n *raft) runAsCandidate() {
 				if n.wonElection(votes) {
 					// TODO If this server was also leader in n.term-1, then we could skip the timer as well.
 					// This would be ok as we'd be guaranteed to have the latest history.
-					if len(n.peers) == votes {
+					if len(n.peers) == votes || n.lxfer {
```

Comment:
I mentioned to Matthias that I feel that this addition "negates" his previous fix, so I am not sure about this change..

Reply:
I am a hard no on this.

Reply:
This is when a current leader with quorum selects us to be the new leader, in that case there is no reason to wait. IMO the system is painful to work with when you just ctrl-c a server without this. If we have lost quorum and do not have a leader Matthias code will apply..